src/Sylius/Behat/Context/Setup/TaxRateContext.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Setup;
+
+final class TaxRateContext
```

Comment:
Do we need this file?

Reply:
Idk how did I push that
![image](https://user-images.githubusercontent.com/80641364/220452492-9a11f482-e1a6-4bef-976c-7d101d5da2ec.png)